server/neptune/gateway/event/comment_handler.go

```diff
@@ -68,18 +79,47 @@ func (p *CommentEventWorkerProxy) Handle(ctx context.Context, request *http.Buff
 		return p.forwardToSns(ctx, request)
 	}
 
-	if shouldAllocate && command.ForceApply {
-		p.logger.InfoContext(ctx, "running force apply command")
-		if commentErr := p.vcsClient.CreateComment(event.BaseRepo, event.PullNum, warningMessage, ""); commentErr != nil {
-			p.logger.ErrorContext(ctx, commentErr.Error())
+	if shouldAllocate {
+		if cmd.ForceApply {
+			p.logger.InfoContext(ctx, "running force apply command")
```

Comment:
Does this log have tags that are helpful when debugging? In general lets add tags to every log where possible to give as much context, you will thank yourself later should you ever need to debug this.

Reply:
Yup, we use a logger extractor to add tags from the context, so that we don't have to add tags everywhere. https://github.com/lyft/atlantis/blob/14806571e8df56e573d57ba9fe12dcd1a9c271af/server/logging/logger.go#L58-L63
Also validated that it works in this [log](https://kibana-staging.lyft.net/app/discover/#/doc/cf3dd9e0-7305-11eb-bad9-93799307c46d/elasticlogsnt3:service-001107?id=4e1345f6-07b9-4573-aca7-a4dcf46200ec)